src/main/java/com/cincialert/dto/User.java

```diff
@@ -1,4 +1,4 @@
-package com.cincialert.model;
+package com.cincialert.dto;
```

Comment:
your DTO object was stored in a "model" folder. Although this is okay, standard development practices include the models in the "dto".